app/javascript/src/components/Dashboard/Contacts/Table/index.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+import { Table as NeetoTable } from "neetoui";
```

Comment:
It should be `NeetoUI`. Refer: https://academy.bigbinary.com/learn-react/react-best-practices/miscellaneous-tips#:~:text=Importing%20under%20an%20alias